packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -116,8 +117,12 @@ export class ApolloServerBase<Request = RequestInit> {
 
     if (requestOptions.persistedQueries !== false) {
       if (!requestOptions.persistedQueries) {
+        //maxSize is the number of elements that can be stored inside of the cache
+        //https://github.com/withspectrum/spectrum has about 200 instances of gql`
+        //300 queries seems reasonable
+        const lru = new QuickLru({ maxSize: 300 });
         requestOptions.persistedQueries = {
-          cache: new Keyv(),
+          cache: new Keyv({ store: lru }),
```

Comment:
Do we need `Keyv` here? I think we could just use `QuickLru` as the cache implementation directly.

Reply:
Keyv adds the ability to have the same Promise based interface